tests/hypothesis_strategies/alternative_creation/test_kernels.py

```diff
@@ -0,0 +1,40 @@
+"""Test alternative ways of creation not considered in the strategies."""
+
+from fractions import Fraction
+
+import hypothesis.strategies as st
+import pytest
+from hypothesis import given
+
+from baybe.kernels import MaternKernel
+
+# Strategies for creating (hopefully different) representations of the fractions that
+# can be used to create the MaternKernel
+one_half_fraction_strategy = st.fractions(min_value=0.5, max_value=0.5)
+three_halves_fraction_strategy = st.fractions(min_value=1.5, max_value=1.5)
+five_halves_fraction_strategy = st.fractions(min_value=2.5, max_value=2.5)
+
+# Putting all of the fraction strategies together
+fraction_strategy = st.one_of(
+    one_half_fraction_strategy,
+    three_halves_fraction_strategy,
+    five_halves_fraction_strategy,
```

Comment:
as this will shrink towards the first entry I would put the 5/2 first as its our default

Reply:
Will change if we decide to keep this.

Reply:
Resolved as there is another thread to discuss this test.